packaging/dbscripts/upgrade/pre_upgrade/0000_config.sql

```diff
@@ -647,7 +647,7 @@ select fn_db_add_config_value('VdsRefreshRate','2','general');
 select fn_db_add_config_value('vdsRetries','0','general');
 select fn_db_add_config_value('vdsTimeout','180','general');
 select fn_db_add_config_value('WindowsGuestAgentUpdateCheckInternal', '180', 'general');
-select fn_db_add_config_value('VdsCertificateValidityInDays','398','general');
+select fn_db_add_config_value('VdsCertificateValidityInDays','3650','general');
```

Comment:
after more discussion we're leaning towards 5y

Reply:
Is there any reasoning we can add to the changed value?

Reply:
just the fact that 5y is what it always used to be. More is generally discouraged as a best practice (even 5 is discouraged, but again, this is waht we had and everyone was fine with it)

Reply:
There used to be an argument that 10 years is very likely to be more than the hardware lifetime so in such a case the certificate update wouldn't be needed at all in most cases.  But OK, we used to have 5 years, let it be 5 years then.